freedomofpress/securedrop-debian-packaging
# Build wheels with build

# Description

Resolves https://github.com/freedomofpress/securedrop-debian-packaging/issues/239

You can see there are 4 changes here:

* We now allow all devs on the team to sign wheels. You can see that our pubkeys have been added in the first commit.
* There are three new wheels. The versions are the same but there is a variation between packages that link to .so files that are built with `build` (our new packaging tool) versus `pip`. @conorsch plz verify or correct me if I am mistaken.
* The updated sha256sums file with my signature this time
* A small fix to support projects with a requirments directory. Without this a new `build-requirements.txt` file is created in `securedrop-client` when it already exists in `requirements/build-requirements.txt`.

# Test Plan

- Check out https://github.com/freedomofpress/securedrop-client/pull/1238 for the client and https://github.com/freedomofpress/securedrop-proxy/pull/85 for the proxy and this PR branch
- Run through steps listed here: https://github.com/freedomofpress/securedrop-debian-packaging/issues/239
- [ ] Confirm that there are no new requirements files generated for client, proxy, and log
- [ ] Build the debian packages to confirm we are prepared to successfully release in the future without error
